api/v1/User.php

```diff
@@ -28,6 +28,43 @@ final public function __construct()
     {
     }
 
+    // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
+    public function CheckGameSessionPasswords(): array
+    {
+        $adminHasPassword = true;
+        $playerHasPassword = true;
+        $connection = ConnectionManager::getInstance()->getCachedServerManagerDbConnection();
+        $qb = $connection->createQueryBuilder();
+        $qb->select('password_admin', 'password_player')
+            ->from('game_list')
+            ->where($qb->expr()->eq('id', $this->getGameSessionId()));
+        $passwordData = $connection->executeQuery($qb->getSQL())->fetchAllAssociative();
+        
+        if (!parent::isNewPasswordFormat($passwordData[0]["password_admin"])
+            || !parent::isNewPasswordFormat($passwordData[0]["password_player"])
+        ) {
+            $adminHasPassword = !empty($passwordData[0]["password_admin"]);
+            $playerHasPassword = !empty($passwordData[0]["password_player"]);
```

Comment:
This was logging using the session database using :
```
        $connection = ConnectionManager::getInstance()->getCachedGameSessionDbConnection($this->getGameSessionId());
        $this->CheckVersion($build_timestamp);
        $qb = $connection->createQueryBuilder()
            ->select('game_session_password_admin', 'game_session_password_player')
            ->from('game_session');
        $passwords = $connection->executeQuery($qb->getSQL())->fetchAllAssociative();
        $password_admin = $passwords[0]["game_session_password_admin"];
        $password_player = $passwords[0]["game_session_password_player"];
```

And I am still using this way of logging in . So lets check what your idea is switching to the server manager side completely . Before I merge again :)